pages/common/gitea.md

```diff
@@ -0,0 +1,29 @@
+# gitea
+
+> Administer Gitea, a lightweight Git hosting server.
+> Requires a configured `app.ini` file or environment variables.
+> More information: <https://docs.gitea.com/administration/command-line>.
+
+- Run the Gitea web server using the default configuration:
+
+`gitea web`
+
+- Create the necessary database schema and tables:
+
+`gitea migrate`
+
+- Run built-in admin commands (like user management):
```

Comment:
Make this more explicit. Describe the command given below instead of an implicit "read the manual for more information".

Reply:
I was thinking more in a sense of giving the users the most commonly used admin commands and then describing what the command does and why you would use it. This'll do though. You can always make another PR to add to the page.